### ISSUE TITLE
bratsDiceLoss Problem

### ISSUE THREAD

**Issue by zenith779** [RobinBruegger/PartiallyReversibleUnet#4]
Dear @RobinBruegger ,

Thanks for making the great work publically available.

But I have a confusion about your loss function. Why the bratsDiceLoss return `(wtLoss + tcLoss + etLoss) / 5` in the `bratsUtils.py` ?  I do not know why you divide by 5 in your code.

Best,
ZongKang

**Comment by RobinBruegger** [comment]
Hi @JiangZongKang,

This should probably have been `(wtLoss + tcLoss + etLoss) / 3`. I do this to compensate for the fact that I am adding 3 loss terms together. Since the division is just a linear scaling of the loss, it does really matter anyways, and can be compensated with the learning rate. 

Best regards,
Robin

**Comment by zenith779** [comment]
Hello @RobinBruegger,

Thanks for your reply.  How to compensate loss with the learning rate?

I have also a problem. The `bratsDiceLossOriginal5`  loss function is used with 5 labels' BraTS 2015 dataset?

Best,
ZongKang

**Comment by RobinBruegger** [comment]
Well, since the loss is divided by 5, one might compensate that by specifying a learning rate that is 5 times bigger. In my experience however, this is not so important when training with the Adam optimizer. 

I used the `bratsDiceLossOriginal5` loss function when I experimented with networks that had the original 5 classes of the dataset as output (and not the nested regions Whole Tumor, Tumor Core and Enhancing Tumor).
